lib/api/Cargo.toml

```diff
@@ -166,6 +168,7 @@ tar = "0.4.42"
 ureq = "2.10.1"
 xz = "0.1.0"
 zip = "2.2.0"
+which = "7.0.0"
```

Comment:
Should this be an optional dependency, only enabled for the backends that require it?

Reply:
[0a0324e](https://github.com/wasmerio/wasmer/pull/5243/commits/0a0324e7cfc31dbbb2a1d3817409b630fc45bf9a)